### PR TITLE
Add social score

### PR DESCRIPTION
**Pull request by lindaicing**
Added social score and changed styling
Screen Shot 2020-08-07 at 8 12 34 PM

### CODE REVIEW

File: Extension/popup.js [hunk 1]
```diff
@@ -27,9 +27,14 @@ function loadExtension(ethicliStats) {
             fullheight= fullheight-50;
             document.getElementById("animalSection").style="display:none;";
         }
+        if (ethicliStats.socialScore == 0.0) {
+            fullheight= fullheight-50;
```
Comment (CubeOfCheese) [inline Extension/popup.js:31]: This code styling kinda wonky

File: Extension/stylesheets/stylesheet.css [hunk 2]
```diff
@@ -273,27 +277,32 @@ body {
...
     align-self: flex-end;
-    padding: 8px;
+    padding: 6px;
+    font-size: 12px;
 }
 
+/* #detailsButton a{
+    font-size: 8px;
+} */
+
```
Comment (CubeOfCheese) [inline Extension/stylesheets/stylesheet.css:305]: Remove

File: Extension/popup.js (hunk 1 above)
Comment (lindaicing) [reply Extension/popup.js:31]: What's up?

Comment (lindaicing) [reply Extension/popup.js]: Fixed

File: Extension/popup.js [hunk 3]
```diff
@@ -14,22 +14,27 @@ function loadExtension(ethicliStats) {
...
             document.getElementById("laborSection").style="display:none;";
         }
         if (ethicliStats.animalsScore == 0.0) {
-            fullheight= fullheight-50;
+            fullheight = fullheight-50;
             document.getElementById("animalSection").style="display:none;";
         }
+        if (ethicliStats.socialScore == 0.0) {
+            fullheight = fullheight-50;
+            document.getElementById("socialSection").style="display:none;";
```
Comment (CubeOfCheese) [inline Extension/popup.js:32]: `document.getElementById("socialSection").style = "display:none;";`
Can we get this formatting (spacing around equal sign)? I know these are old so I guess I missed them in the past, but just noticed.

File: Extension/popup.html [hunk 4]
```diff
@@ -78,13 +78,23 @@ <h6 id="blmsupport" class="tooltip">
                         <p><span id="animalScore"></span> /10</p>
                     </div>
                 </div>
+                <div class="label" id="socialSection">
+                    <p>Social Involvement</p>
+                    <div>
+                        <span>
+                        <span class="scorebar"></span>
+                        <span class="scorebarFill" id="socialScoreBar"></span>
+                        </span>
```
Comment (CubeOfCheese) [inline Extension/popup.html:87]: Why are there these outer spans?

Comment (CubeOfCheese) [reply Extension/popup.html:87]: The other categories had it too so I approved. If they aren't necessary please remove and push to this branch before merging

**Pull request merged**